Sources/Sentry/include/SentryIOTrackingIntegration.h

```diff
@@ -0,0 +1,9 @@
+#import "SentryIntegrationProtocol.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryIOTrackingIntegration : NSObject <SentryIntegrationProtocol>
```

Comment:
`l`: I think the name should contain File.
```suggestion
@interface SentryFileIOTrackingIntegration : NSObject <SentryIntegrationProtocol>
```